Library/Homebrew/test/test_options.rb

```diff
@@ -56,7 +56,7 @@ def test_equality
 
 class OptionsTests < Homebrew::TestCase
   def setup
-    @options = Options.new
+    @options = Options.create
```

Comment:
What's the advantage of that? Feels better to move the logic into the current constructor.